tools/downloader/list_topologies.yml

```diff
@@ -3286,3 +3286,123 @@ topologies:
+      - name: mobilenet-v2.pth
+        sha256: ecbe2b568c8602549fa9e1d5833c63848f490a48d92e5d224d1eb2063e152cf8
+        size: 14205652
+        source:
+          $type: google_drive
+          id: 1jlto6HRVD3ipNkAl1lNhDbkBp7HylaqR
+    output: "classification/mobilenet/v2/pytorch/"
+    framework: pytorch
+    pytorch_to_onnx:
+      - --model-name=mobilenet-v2
+      - --model-path=$dl_dir
+      - --weights=$dl_dir/mobilenet-v2.pth
+      - --input-shape=1,3,224,224
+      - --output-file=$conv_dir/mobilenet-v2.onnx
+      - --input-names=data
+      - --output-names=prob
+    model_optimizer_args:
+      - --model_name=mobilenet-v2
+      - --framework=onnx
+      - --data_type=FP32
+      - --input=data
+      - --mean_values=data[123.675,116.28,103.53]
+      - --scale_values=data[58.624,57.12,57.375]
+      - --output=prob
+      - --input_model=$conv_dir/mobilenet-v2.onnx
+    license: https://raw.githubusercontent.com/tonylins/pytorch-mobilenet-v2/master/LICENSE
+
+  - name: "googlenet-v3-pytorch"
+    description: >-
+      Inception v3 is image classification model pretrained on ImageNet dataset. This
+      is PyTorch implementation of architecture described in the paper "Rethinking
+      the Inception Architecture for Computer Vision" <https://arxiv.org/pdf/1512.00567.pdf> in
+      TorchVision package <https://github.com/pytorch/vision>.
+
+      The model input is a blob that consists of a single image of "1x3x299x299"
+      in RGB order. 
+
+      The model output is typical object classifier for the 1000 different classifications
+      matching with those in the ImageNet database.
+    task_type: "classification"
+    files:
+      - name: inception_v3_google-1a9a5a14.pth
+        sha256: 1a9a5a14f40645a370184bd54f4e8e631351e71399112b43ad0294a79da290c8
+        size: 108857766
+        source: https://download.pytorch.org/models/inception_v3_google-1a9a5a14.pth
+    output: "classification/googlenet/v3/pytorch"
+    framework: pytorch
+    pytorch_to_onnx:
+      - --model-name=inception-v3
+      - --weights=$dl_dir/inception_v3_google-1a9a5a14.pth
+      - --input-shape=1,3,299,299
+      - --output-file=$conv_dir/googlenet-v3.onnx
+      - --input-names=data
+      - --output-names=prob
+    model_optimizer_args:
+      - --model_name=googlenet-v3
+      - --framework=onnx
+      - --data_type=FP32
+      - --input=data
+      - --mean_values=data[127.5]
+      - --scale_values=data[127.5]
+      - --output=prob
+      - --input_model=$conv_dir/googlenet-v3.onnx
+    license: https://raw.githubusercontent.com/pytorch/vision/master/LICENSE
+
+  - name: "resnet-50-pytorch"
+    description: >-
+      Resnet 50 is image classification model pretrained on ImageNet dataset. This
+      is PyTorch implementation based on architecture described in paper Deep Residual
+      Learning for Image Recognition" <https://arxiv.org/pdf/1512.03385> in TorchVision
+      package <https://github.com/pytorch/vision>
```

Comment:
The description doesn't seem to match the original Markdown text - there's a missing quote and no period at the end.